protocol/genesis/ssv/validator/validator.go

```diff
@@ -39,7 +42,7 @@ type Validator struct {
 	Queues  map[genesisspectypes.BeaconRole]queueContainer
 
 	// dutyIDs is a map for logging a unique ID for a given duty
-	dutyIDs *hashmap.Map[genesisspectypes.BeaconRole, string]
+	//dutyIDs *hashmap.Map[genesisspectypes.BeaconRole, string]
```

Comment:
Can we delete it? If not, please leave a comment why we need to keep it

Reply:
removing